instrumentation/jms/jms-3.0/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/jms/v3_0/Jms3InstrumentationTest.java

```diff
@@ -71,7 +71,7 @@ static void setUp() throws JMSException {
         new GenericContainer<>("quay.io/artemiscloud/activemq-artemis-broker:artemis.2.27.0")
             .withEnv("AMQ_USER", "test")
             .withEnv("AMQ_PASSWORD", "test")
-            .withEnv("AMQ_EXTRA_ARGS", "--disable-persistence")
+            .withEnv("JAVA_TOOL_OPTIONS", "-Dbrokerconfig.maxDiskUsage=-1")
```

Comment:
should it be `-1` or `100` or doesn't matter?

Reply:
-1 disables the check (https://activemq.apache.org/components/artemis/documentation/latest/configuration-index.html):

> The max percentage of data we should use from disks. The broker will block while the disk is full. Disable by setting -1.